queue/lib/txgh-queue/error_handlers/transifex.rb

```diff
@@ -0,0 +1,23 @@
+require 'txgh'
+
+module TxghQueue
+  module ErrorHandlers
+    class Transifex
+      ERROR_CLASSES = {
+        Txgh::TransifexApiError          => Status.retry_with_delay,
+        Txgh::TransifexNotFoundError     => Status.fail,
+        Txgh::TransifexUnauthorizedError => Status.fail
+      }
+
+      class << self
+        def can_handle?(error_or_response)
+          ERROR_CLASSES.any? { |klass, _| error_or_response.class == klass }
```

Comment:
I would either use `ERROR_CLASSES.key?(error_or_response)` or, if you want to support inheritance, 
`ERROR_CLASSES.any? { |klass, _| error_or_response <= klass }`